designer/client/src/components/AddProcessForm.tsx

```diff
@@ -89,14 +90,13 @@ export function AddProcessForm({
                                 Icon={RequestResponseIcon}
                                 active={value.processingMode === ProcessingMode.requestResponse}
                             />
-                            {/*TODO: Uncomment it when batch processing mode ready*/}
-                            {/*<CustomRadio*/}
-                            {/*    disabled={processingModes.every((processingMode) => processingMode !== ProcessingMode.batch)}*/}
-                            {/*    label={t("addProcessForm.label.batch", "Batch")}*/}
-                            {/*    value={ProcessingMode.batch}*/}
-                            {/*    Icon={BatchIcon}*/}
-                            {/*    active={value.processingMode === ProcessingMode.batch}*/}
-                            {/*/>*/}
+                            <CustomRadio
```

Comment:
It is already uncommented and changed slightly on the release/1.14 branch. I think that we should synchronize branches to avoid conflicts.

Reply:
I've talked to RM and the sync should be done tomorrow - I'll wait till then

Reply:
I already prepared port PR: https://github.com/TouK/nussknacker/pull/5791/files . Let's merge it and we can move forward. I gave you approve in advance